src/lib/Queue.ts

```diff
@@ -66,7 +66,9 @@ class QueueImpl<T> implements IQueue<T>, Iterable<T> {
    * @param entry
    */
   public includes(entry: T) {
-    return this.entries.includes(entry);
+    return this.entries.some(
+      (currentEntry) => JSON.stringify(currentEntry) === JSON.stringify(entry)
+    );
```

Comment:
In the context of `FlashMessagesController` this is fine, since we control how we create queue entries. But this is a generic class that should be usable in other places, as well. See this example:

![image](https://user-images.githubusercontent.com/13508038/140789553-b875bb41-d1d5-4548-ba85-54f32c814f65.png)

The objects should be equal, but they're not, since the keys are in a different order.

We could use a deep comparison algorithm here. We already use `underscore`'s `isEqual()` in another place, I think it would be better to use it here, as well.